ruler-frontend/src/main/kotlin/com/spotify/ruler/frontend/components/Ownership.kt

```diff
@@ -123,3 +137,28 @@ fun RBuilder.highlightedValue(value: Number, label: String, formatter: NumberFor
         span(classes = "text-muted m-0") { +label }
     }
 }
+
+private fun getSizesByOwner(components: List<AppComponent>): Map<String, Measurable> {
```

Comment:
Refactored as suggested [here](https://github.com/spotify/ruler/pull/102#discussion_r1028248027) in order to improve readability